modules/ingestor/src/service/format.rs

```diff
@@ -120,40 +130,60 @@ impl<'g> Format {
     }
 
     pub fn is_csaf(bytes: &[u8]) -> Result<bool, Error> {
-        Ok(masked(
+        match masked(
             key("document").and(key("csaf_version")).and(depth(2)),
             bytes,
-        )?
-        .is_some())
+        ) {
+            Ok(Some(_)) => Ok(true),
+            _ => Ok(false),
```

Comment:
The `_` prevents me from understanding why these changes to all the `is_*` fn's that call `masked` are required without any changes to `masked` itself. Have these always been wrong?

Reply:
You'd prefer the explicit `Err(_) | Ok(None)` instead of the catch-all? I'm cool with that also.